_layouts/timeline.html

```diff
@@ -24,7 +25,7 @@
 {%- if navYears -%}
 <div class="dropdown float-end" id="year-nav">
     <button class="btn btn-primary dropdown-toggle" type="button" id="yearButton" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-        Year
+        years
```

Comment:
```suggestion
        {{ year }}
```
?
but is it good idea to translate variables within liquid code?